mitosheet/src/mito/components/endo/ColumnHeader.tsx

```diff
@@ -82,7 +82,8 @@ const ColumnHeader = (props: {
     // Get the pieces of the column header. If the column header is not a MultiIndex header, then
     // lowerLevelColumnHeaders will be an empty array
     const { lowerLevelColumnHeaders, finalColumnHeader } = getColumnHeaderParts(columnHeader);
-    const borderStyle = getBorderStyle(props.gridState.selections, props.gridState.copiedSelections, -1, props.columnIndex, props.sheetData.numRows, props.uiState.highlightedColumnIndex);
+    const matchesSearch = !!props.uiState.currOpenSearch.searchValue && (finalColumnHeader + '').toLowerCase().includes(props.uiState.currOpenSearch.searchValue.toLowerCase());
```

Comment:
Wanna factor all of this logic into a function, and share it between column header and grid data?